subsys/usb/usb_c/usbc_prl.c

```diff
@@ -47,6 +47,8 @@ enum prl_flags {
 	 * cleared when the message is sent to the TCPC layer.
 	 */
 	PRL_FLAGS_MSG_XMIT = 7,
+	/** Flag to track if first message in AMS is pending */
+	PRL_FLAGS_FIRST_MSG_PENDING = 8,
```

Comment:
Commit [usb-c: Properly detect reception of first AMS message](https://github.com/zephyrproject-rtos/zephyr/pull/52993/commits/1e9fb5a23ad27ca86715b9bb79c4ec577ebe23bd) commit needs to be the parent of the [usb-c: Refactor soft reset code](https://github.com/zephyrproject-rtos/zephyr/pull/52993/commits/a616f11e92a0ef7f476f0c7af77f692f85d7562e)

Reply:
done